recipes/sire/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: sire
+  version: 2018.2.0_300_ge77b675
+
+source:
+  url: https://objectstorage.eu-frankfurt-1.oraclecloud.com/p/WaA-Qcb9K3rm_E6VeekU_gjAz3bSRH9zTVp6cTsGbcE/n/chryswoods/b/sire_releases/o/sire_conda_latest_linux.tar.bz2  # [linux]
```

Comment:
Are you just downloading the compiled binaries and copying it?
It should be good to download the source code and compile it using our compilers to avoid any kind of abi problems

Reply:
We would like to be able to build Sire on your platform, but as mentioned above the build can take several hours and we have run into issues on Travis, e.g. time-outs. We didn't want to place undue burden on your system.

Sire is built in a miniconda environment using the Conda compilers so there shouldn't be a problem there. We also build against specific versions of Linux and macOS to ensure the widest possible compatibility for our binaries.